packages/kit/src/exports/vite/index.js

```diff
@@ -695,8 +695,13 @@ Tips:
 								assetFileNames: `${prefix}/assets/[name].[hash][extname]`,
 								hoistTransitiveImports: false,
 								sourcemapIgnoreList,
-								manualChunks: split ? undefined : () => 'bundle',
-								inlineDynamicImports: false
+								manualChunks:
+									// @ts-ignore `vite.rolldownVersion` only exists in `rolldown-vite`
+									vite.rolldownVersion || split ? undefined : () => 'bundle',
+								inlineDynamicImports:
+									split &&
+									// @ts-ignore `vite.rolldownVersion` only exists in `rolldown-vite`
+									!!vite.rolldownVersion
```

Comment:
It might be nice to just have `const isRolldown = !!vite.rolldownVersion;` up above a single time so that the ignore doesn't have to be repeated everywhere

I think `ts-expect-error` would be more appropriate to use here. I'm curious why you switched it

Reply:
> It might be nice to just have `const isRolldown = !!vite.rolldownVersion;` up above a single time so that the ignore doesn't have to be repeated everywhere

Oh good idea. I'll add that.

> I think `ts-expect-error` would be more appropriate to use here. I'm curious why you switched it

@dominikg mentioned it would result in errors for the rolldown-vite ecosystem CI since it installs `rolldown-vite` and that means the property does exist in these cases (`@ts-expect-error` will error if it doesn't find any error)